pyproject.toml

```diff
@@ -15,7 +15,6 @@ force-exclude = '''
   | setup.py
 )
 '''
-check = true
```

Comment:
This line makes it so you can't just `black pyvistaqt/plotting.py` for example to actually fix the files. I think whatever utility/CI actually wants to run just a check should use `--check` itself rather than having this live in `pyproject.toml`